### PR TITLE
fix: reenable shadows for blocks being dragged

### LINKED ISSUE #59: Blocks are missing shadows during drags
While being dragged, Scratch adds drop shadows to blocks:

Screenshot 2024-05-06 at 10 49 04 AM (2)

These need to be added.

### PR DESCRIPTION
**Pull request by gonfunko**
This PR adds a drop shadow to blocks that are being dragged. Fixes #59.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -60,12 +61,27 @@ export function inject(container, options) {
   const workspace = Blockly.inject(container, options);
   workspace.getRenderer().getConstants().selectedGlowFilterId = '';
 
+  // Add a drop shadow to mid-drag blocks.
+  workspace.addChangeListener((event) => {
+    if (event.type === Blockly.Events.BLOCK_DRAG) {
+      if (event.isStart) {
+        event.blocks[0].getSvgRoot().setAttribute(
```
Comment (BeksOmega) [inline src/index.js]: Can you set filters via css? If so, then can you use the `blocklyDragging` class instead?

That class does apply to workspace comments as well so you might have to check for `not(.blocklyComment)`, until we actually add a class for blocks.

File: src/shadows.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import * as Blockly from 'blockly/core';
```
Comment (BeksOmega) [inline src/shadows.js:7]: Just realized I haven't been checking this super closely, but this file probably needs a license?

File: src/shadows.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  const svg = workspace.getParentSvg();
+  const defs = Blockly.utils.dom.createSvgElement(Blockly.utils.Svg.DEFS, {}, svg);
+  // Adjust these width/height, x/y properties to stop the shadow from clipping
+  var dragShadowFilter = Blockly.utils.dom.createSvgElement('filter',
+      {
+        'id': 'blocklyDragShadowFilter',
+        'height': '140%',
+        'width': '140%',
+        'y': '-20%',
+        'x': '-20%'
```
Comment (BeksOmega) [inline src/shadows.js:20]: I am so confused but ok haha.

File: src/index.js (hunk 1 above)
Comment (gonfunko) [reply src/index.js]: Ooh yes, and workspace comments also have drop shadows, so that's a plus actually.

File: src/shadows.js (hunk 2 above)
Comment (gonfunko) [reply src/shadows.js:7]: Done

**Review by BeksOmega — APPROVED** [review]

**Pull request merged**